tests/js/spec/views/usageStats/index.spec.jsx

```diff
@@ -3,34 +3,36 @@ import React from 'react';
 import {mountWithTheme} from 'sentry-test/enzyme';
 import {initializeOrg} from 'sentry-test/initializeOrg';
 
+import {DataCategory} from 'app/types';
 import UsageStats from 'app/views/usageStats';
+import {CHART_OPTIONS_DATA_TRANSFORM} from 'app/views/usageStats/usageChart';
 
-describe('UserFeedback', function () {
-  const {organization, routerContext} = initializeOrg();
-  const orgSlug = organization.slug;
+describe('UsageStats', function () {
+  const router = TestStubs.router();
+  const {organization, routerContext} = initializeOrg({router});
 
-  const orgUrl = `/organizations/${orgSlug}/stats_v2/`;
-  const projectUrl = `/organizations/${orgSlug}/stats_v2/projects/`;
+  const orgUrl = `/organizations/${organization.slug}/stats_v2/`;
+  const projectUrl = `/organizations/${organization.slug}/stats_v2/projects/`;
```

Comment:
not sure if this url is correct -- ended up combining all the routes into one, and to group by project we'd do `groupBy=project` in the url query.

Reply:
This is fixed in https://github.com/getsentry/sentry/pull/24953